pkg/cmd/openshift-controller-manager/openshiftcontrolplane_default.go

```diff
@@ -3,14 +3,25 @@ package openshift_controller_manager
 import (
 	"time"
 
+	"k8s.io/klog"
+
+	configv1 "github.com/openshift/api/config/v1"
 	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
 	"github.com/openshift/library-go/pkg/config/configdefaults"
 	"github.com/openshift/library-go/pkg/config/helpers"
 	leaderelectionconverter "github.com/openshift/library-go/pkg/config/leaderelection"
 )
 
 func setRecommendedOpenShiftControllerConfigDefaults(config *openshiftcontrolplanev1.OpenShiftControllerManagerConfig) {
-	configdefaults.SetRecommendedHTTPServingInfoDefaults(config.ServingInfo)
+	if config.ServingInfo == nil {
+		// this condition  will be satisfied when there no "servingInfo" entry is in the configuration file
+		config.ServingInfo = &configv1.HTTPServingInfo{}
+		configdefaults.SetRecommendedHTTPServingInfoDefaults(config.ServingInfo)
+	} else if config.ServingInfo != nil && len(config.ServingInfo.BindAddress) == 0 {
+		// this condition will be satisfied when there is empty "servingInfo": {} entry in the configuration file
```

Comment:
I meant more high level: `// disable serving if already set with empty bind address, used in dev setup`

Reply:
okay, updated - https://github.com/openshift/openshift-controller-manager/pull/30/commits/56c7e6f3d62986879b56bb416ecf3e3ad606359b